client/store/ui/notification/stateType.ts

```diff
@@ -0,0 +1,10 @@
+export interface NotificationMessage {
+  duration?: number
+  id: string
```

Comment:
how do you manage id when creating new toaster?
ex. on ajax error?

would it be simpler to use array index?